signed/interface.go

```diff
@@ -0,0 +1,20 @@
+package signed
+
+import (
+	"github.com/endophage/go-tuf/data"
+	"github.com/endophage/go-tuf/keys"
+)
+
+type SigningService interface {
+	Sign(keyIDs []string, data []byte) ([]data.Signature, error)
```

Comment:
Is passing in multiple key IDs intended as an optimization when signing data with multiple keys?